src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,7 +19,7 @@
  */
 public class RobotContainer {
 
-    public static TurretSubsystem TURRET = new TurretSubsystem(
+    public static Turret TURRET = new Turret(
```

Comment:
In the FTC you said to do it turret (lower case), choose one
Also make it final